File: bench/src/go/pi.go
package main
import "os"
func main(){ pi:=0.0; sign:=1.0
  for i:=uint64(0);i<20000000;i++ { d:=2*i+1; pi += sign/float64(d); sign=-sign }
  scaled:=pi*4000000000.0; check:=uint64(scaled); os.Exit(int(check%256)) }

File: bench/src/rust/pi.rs
fn main(){
  let mut pi=0.0f64; let mut sign=1.0f64;
  for i in 0u64..20000000 { let d=2*i+1; pi += sign/(d as f64); sign=-sign; }
  let scaled=pi*4000000000.0; let check=scaled as u64; std::process::exit((check%256) as i32);
}
